Modelica_DeviceDrivers/Resources/test/Communication/test_MDDMQTT_buildAsSharedLibrary.c

```diff
@@ -0,0 +1 @@
+#include "../../Include/MDDMQTT.h"
```

Comment:
One more thought: Is this actually enough? Or should there be some implementation?

Reply:
Think it should be enough to catch the error. I tested once with the old libraries and got the expected error, when I tested with your new commit and it worked.

Reply:
OK, that's the information I was missing. Thanks.

Reply:
It won't give an error during running the tests, it either fails or passes during compilation. However, a build fail is detected by travis and so we get the information.